lib/services/messenger/translators/front.ts

```diff
@@ -335,7 +335,7 @@ export class FrontTranslator extends TranslatorScaffold implements Translator {
 			const messageString = TranslatorScaffold.convertPings(message.details.text, converter);
 			const createThreadData: MessageRequest.Send = {
 				author_id: userId,
-				body: `${messageString}<br />${metadataString}`,
+				body: `${marked(messageString.replace(/\n+/g, '\n\n'))}<br />${metadataString}`,
```

Comment:
This is slightly nuclear :) “messageString” can potentially include preformatted text that should stay intact and this blindly replaces newlines in it. I think leaving it to “marked” is a better option.